ruby-test-mode.el

```diff
@@ -69,25 +70,45 @@ Test Driven Development in Ruby."
   "The keymap used in `ruby-test-mode' buffers.")
 
 (defcustom ruby-test-file-name-extensions
-  '("builder" "erb" "haml" "rb" "rjs")
+  '("builder" "erb" "haml" "rb" "rjs" "rake" "slim")
   "*A list of filename extensions that trigger the loading of the
 minor mode."
   :type '(list)
   :group 'ruby-test)
 
 (defcustom ruby-test-implementation-filename-mapping
   '(
```

Comment:
I think you need to backquote here, otherwise you get won't get the translated regex.

``` elisp
(pp '(((pcre-to-elisp "(.*)/spec/routing/routes_spec\\.rb$") "\\1/config/routes.rb")
      ((pcre-to-elisp "(.*)/test/routing/routes_test\\.rb$") "\\1/config/routes.rb")))

(((pcre-to-elisp "(.*)/spec/routing/routes_spec\\.rb$")
  "\\1/config/routes.rb")
 ((pcre-to-elisp "(.*)/test/routing/routes_test\\.rb$")
  "\\1/config/routes.rb"))
```

I think what you want is something like this:

``` elisp
(pp `((,(pcre-to-elisp "(.*)/spec/routing/routes_spec\\.rb$") "\\1/config/routes.rb")
      (,(pcre-to-elisp "(.*)/test/routing/routes_test\\.rb$") "\\1/config/routes.rb")))

(("\\(.*\\)/spec/routing/routes_spec\\.rb$" "\\1/config/routes.rb")
 ("\\(.*\\)/test/routing/routes_test\\.rb$" "\\1/config/routes.rb"))
```